site/src/Consumer/WebNotificationConsumer.php

```diff
@@ -15,27 +18,78 @@
  */
 class WebNotificationConsumer extends AbstractNotificationConsumer
 {
+
+    /**
+     * @var NotificationManager
+     */
+    protected $notificationManager;
+
+    /**
+     * @var UserManagerInterface;
+     */
+    protected $userManager;
+
     /**
      * WebNotificationConsumer constructor
      *
-     * @param LoggerInterface $logger
+     * @param LoggerInterface     $logger
+     * @param NotificationManager $notificationManager
      */
-    public function __construct(LoggerInterface $logger)
-    {
+    public function __construct(
+        LoggerInterface $logger,
+        NotificationManager $notificationManager,
+        UserManagerInterface $userManager
```

Comment:
this last argument is not documented in the phpdoc.

Reply:
`userManager` is not required in this Consumer, yet